server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bsp/managers/BazelToolchainManager.kt

```diff
@@ -3,12 +3,17 @@ package org.jetbrains.bsp.bazel.server.bsp.managers
 import org.eclipse.lsp4j.jsonrpc.CancelChecker
 import org.jetbrains.bazel.commons.label.Label
 import org.jetbrains.bsp.bazel.bazelrunner.BazelRunner
+import org.jetbrains.bsp.bazel.server.bsp.managers.Language
 import org.jetbrains.bsp.protocol.FeatureFlags
 
 class BazelToolchainManager(private val bazelRunner: BazelRunner, private val featureFlags: FeatureFlags) {
   fun getToolchain(ruleLanguage: RuleLanguage, cancelChecker: CancelChecker): Label? =
     when (ruleLanguage.language) {
-      Language.Scala -> Label.parse("@io_bazel_rules_scala//scala:toolchain_type")
+      Language.Scala ->
+        when (ruleLanguage.ruleName) {
+          "rules_scala_annex" -> Label.parse("@rules_scala_annex//rules/scala:toolchain_type")
+          else -> Label.parse("@${ruleLanguage.ruleName}//scala:toolchain_type")
```

Comment:
it 
```suggestion
          else -> Label.parse("@io_bazel_rules_scala//scala:toolchain_type")
```

even in bzlmod, it's still prefixed with `io_bazel`